mne/epochs.py

```diff
@@ -1700,7 +1708,7 @@ class EpochsArray(Epochs):
     @verbose
     def __init__(self, data, info, events, tmin=0, event_id=None,
                  reject=None, flat=None, reject_tmin=None,
-                 reject_tmax=None, verbose=None):
+                 reject_tmax=None, baseline=(None, 0), verbose=None):
```

Comment:
yeah, it's a different order. that's one pet-peeve I hate: inconsistent
orderings in similar functions. there are a few other arguments that not in
the same too. we always have preload, verbose in the last positions, right?

-teon

On Sat, Apr 11, 2015 at 11:57 PM, Eric Larson <notifications@github.com>
wrote:

> In mne/epochs.py
> <https://github.com/mne-tools/mne-python/pull/1951#discussion_r28198786>:
>
> > @@ -1700,7 +1708,7 @@ class EpochsArray(Epochs):
> >      @verbose
> >      def __init__(self, data, info, events, tmin=0, event_id=None,
> >                   reject=None, flat=None, reject_tmin=None,
> > -                 reject_tmax=None, verbose=None):
> > +                 reject_tmax=None, baseline=(None, 0), verbose=None):
>
> Ahh, right. The default for this should be None then, otherwise it's an
> API change. It makes it different from Epochs, right? We might have to
> live with that difference so we don't break people's code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/1951/files#r28198786>.
>